firestore/angular-rewrite/README.md

```diff
@@ -19,3 +19,9 @@ Run `ng build` to build the project. The build artifacts will be stored in the `
 ## Further help
 
 To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.
+
+## Running the Emulators
+The command to start the local emulators is:
```

Comment:
Really we care about starting the app, let's provide the exec command with ng serve also.